### DISCUSSION TITLE
No Download URL Found, Skipping

### DISCUSSION THREAD

**Discussion by baxterblk** [RandomNinjaAtk/arr-scripts#104] [Q&A] [answered]
I get the title error within Sonarr when scanning the 1 YT Series I have available. Without naming the series directly, it's the series where people get interviewed while eating spicy hot wings. I'm not sure if there's something I'm supposed to input here or not but it doesn't seem to want to grab anything for this "series". Any information would be appreciated! Thanks in advance. LOVE the work you've done here.

**Comment by RandomNinjaAtk** [comment, marked as answer]
Some reading for you, but the information is mostly found in the legacy repo:

https://github.com/RandomNinjaAtk/docker-sonarr-extended/discussions/28#discussioncomment-5453192
and
https://github.com/RandomNinjaAtk/docker-sonarr-extended/issues/21

If I had to guess the information required for the script to work is not in the database, which would mean someone needs to update the website... So that could be you, and you'd help everyone. Please read through the links above to have the context needed for this comment...

**Reply by baxterblk** [reply]
Great! I'll let you know if I'm able to contribute anything worthy.
